### PR TITLE
Drag n drop

### PR DESCRIPTION
**Pull request by MykytaNesterov**
# Pull Request ☁️
## Overview ✨
Added drag and drop 

## Changes 🔧
Suffered
- __[DialogPlugin]__  _Deleted handleSetDefault() function._
- __[DragableListItem, CreatePluginPage]__  _Added drag and drop for plugin fields._

## Additional context 🌟
Andriy Demchishin owes me 3 hours of wasted time @ZaykaNya

### CODE REVIEW

File: src/entities/InputField.ts [hunk 1]
```diff
@@ -27,6 +27,8 @@ export default class InputField extends BasicPluginField {
      * max - max value
      */
     public max: number;
+
```
Comment (ZaykaNya) [inline src/entities/InputField.ts]: jdocs

File: src/views/CreatePluginPageView/CreatePluginPageView.tsx [hunk 2]
```diff
@@ -51,25 +56,52 @@ const CreatePluginPageView = React.forwardRef((props: CreatePluginPageViewProps,
...
-    function handleDeletePluginField(field: BasicPluginField | InputField) {
+    function handleDeletePluginField(field: InputField) {
         setPluginFields(pluginFields.filter(item => item.name !== field.name));
     }
 
     function handleSetIsDialogPluginButtonActive() {
         setIsDialogPluginButtonActive(true);
     }
 
+    const renderField = (item: InputField, index: number) => {
```
Comment (DanilAndreev) [inline src/views/CreatePluginPageView/CreatePluginPageView.tsx:94]: Move it to jsx

File: src/views/CreatePluginPageView/LocalComponents/DragableListItem/DragableListItem.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+            isDragging: monitor.isDragging(),
+        }),
+    });
+
+    drag(drop(refer));
+
+    const opacity = isDragging ? 0 : 1;
+
+    return(
+            <div ref={refer} style={{opacity}}>
```
Comment (DanilAndreev) [inline src/views/CreatePluginPageView/LocalComponents/DragableListItem/DragableListItem.tsx]: Box instead div!

Comment (DanilAndreev) [reply src/views/CreatePluginPageView/LocalComponents/DragableListItem/DragableListItem.tsx]: !remove div and pass refs to ```<ListItem/>```

**Review by DanilAndreev — CHANGES_REQUESTED** [review]

**Review by DanilAndreev — APPROVED** [review]

**Pull request merged**